### PR TITLE
Create features section

### PR DESCRIPTION
**Pull request by Arthelon**
Addresses #18  
Adds responsive section displaying the 8 feature items. Each feature has an empty image for the time being.

### CODE REVIEW

File: src/components/Features.svelte [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<script>
+  const features = [
+    {
+      title: "Single Sign-On",
+      description: "Login once to multiple applications",
+      imgSrc: "",
```
Comment (louischan-oursky) [inline src/components/Features.svelte]: Empty source makes the page look weird. Maybe use something like https://placeholder.com/ to stub the images first?

Comment (Arthelon) [reply src/components/Features.svelte]: sure, I'll do that

File: src/components/Features.svelte [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+<section class="features-section">
+  <div class="container-fluid">
+    <div class="row">
+      {#each features as feature}
+        <div class="feature-wrapper col-12 col-md-3">
+          <div class="feature-img-wrapper">
+            <img
+              class="feature-img"
+              src={feature.imgSrc || 'https://via.placeholder.com/95'}
```
Comment (kiootic) [inline src/components/Features.svelte]: Can you move the URL to individual feature descriptor object, and add a TODO to remind us of replacing the placeholder?

Comment (Arthelon) [reply src/components/Features.svelte]: yup

**Review by kiootic — APPROVED** [review]

**Pull request merged**